frontend/src/types.ts

```diff
@@ -3857,6 +3857,8 @@ interface BreadcrumbBase {
     key: string | number | [scene: Scene | string, key: string | number]
     /** Whether to show a custom popover */
     popover?: Pick<PopoverProps, 'overlay' | 'matchWidth'>
+    /** Whether to show a custom popover for the project */
+    is_popover_project?: boolean
```

Comment:
1. We use camelCase in types originating in the frontend
2. Sounds like this should be solved with just `popover` for alignment with other breadcrumbs – would be much preferable to `isPopoverProject` being a special snowflake